frontend/app/model/Eventbrite.scala

```diff
@@ -289,6 +290,9 @@ object Eventbrite {
       }
     } yield uri
 
+    val mainImageWithNoCrop: Boolean =
```

Comment:
Given that this is a boolean it'd be nice to name it something that is more obviously a predicate
like mainImageHasNoCrop or something, I'd expect mainImageWithNoCrop to be an uncropped main image